src/libraries/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -187,6 +188,7 @@ public void ValidateDeserializationOfObjectWithGenericTypeWhichGenericArgumentHa
         [Fact]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/34008", TestPlatforms.Linux, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
         [ActiveIssue("https://github.com/dotnet/runtime/issues/34753", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
+        [PlatformSpecific(~TestPlatforms.Browser)]
```

Comment:
@akoeplinger this is one of the tests we would end up disabling after BinaryFormatter is killed on WASM.

Maybe we want to update to use conditional with `PlatformDetecion.IsBinaryFormatterSupported`

Reply:
@safern yeah, feel free to send a follow up PR for that :)